tools/wpt/run.py

```diff
@@ -422,7 +422,7 @@ def setup_kwargs(self, kwargs):
             binary = self.browser.find_binary(self.venv.path, None)
 
             if binary is None:
-                raise WptrunError("Unable to find servo binary on the PATH")
+                raise WptrunError("Unable to find servo binary in PATH")
```

Comment:
Not sure that this was a change important enough to make here.

Reply:
Er, I guess I'll leave it then? Of course it's very easy to remove if desired.

(Trivia: it most likely comes from Spanish, which uses "en" for both "in" and "on," making it rather difficult to pick the right one when writing in English.)